go.mod

```diff
@@ -0,0 +1,5 @@
+module code.google.com/p/gopacket
```

Comment:
Should be enough to have single `go.mod` at `/resources/scripts/source/ipip_check`